app.coffee

```diff
@@ -24,19 +24,19 @@ app.use express.static(path.join(__dirname, 'public'))
 app.set 'mongo url', "mongodb://127.0.0.1:27017/ethics-api?slaveOk=true"
 
 # catch 404 and forward to error handler
-app.use (req, res, next) ->
-  err = new Error('Not Found')
-  err.status = 404
-  next err
+# app.use (req, res, next) ->
+#   err = new Error('Not Found')
+#   err.status = 404
+#   next err
 
 # development error handler
 # will print stacktrace
-if app.get('env') == 'development'
-  app.use (err, req, res, next) ->
-    res.status err.status || 500
-    res.render 'error',
-      message: err.message,
-      error: err
+# if app.get('env') == 'development'
+#   app.use (err, req, res, next) ->
+#     res.status err.status || 500
+#     res.render 'error',
+#       message: err.message,
+#       error: err
```

Comment:
We definitely can't leave these commented out. They're important. What error were you seeing?